config/vehicle_info.yaml

```diff
@@ -1,9 +1,12 @@
-wheel_radius: 0.39
-wheel_width: 0.42
-wheel_base: 2.74 # between front wheel center and rear wheel center
-wheel_tread: 1.63 # between left wheel center and right wheel center
-front_overhang: 1.0 # between front wheel center and vehicle front
-rear_overhang: 1.03 # between rear wheel center and vehicle rear 
-left_overhang: 0.1 # between left wheel center and vehicle left
-right_overhang: 0.1 # between right wheel center and vehicle right
-vehicle_height: 2.5
+/**:
+  ros__parameters:
+
```

Comment:
Nit: Why the newline?

Reply:
copy & paste. I removed it now in all three files